### ISSUE TITLE
Unable to install/use pgroonga extension (Symbol not found: _build_reloptions)

### ISSUE THREAD

**Issue by Somnium99** [pgroonga/pgroonga#151]
I followed the Install Guide. But it didn't work - I got this error :
`ERROR: could not open extension control file "/Applications/Postgres.app/Contents/Versions/12/share/postgresql/extension/pgroonga.control": No such file or directory`

I then tried this solution (manually copying the files) :
https://stackoverflow.com/questions/45736639/installing-pgroonga-on-macosx-with-postgres-app/53070341#53070341 

And now I'm getting this error :
`
ERROR:  could not load library "/Applications/Postgres.app/Contents/Versions/12/lib/postgresql/pgroonga.so": dlopen(/Applications/Postgres.app/Contents/Versions/12/lib/postgresql/pgroonga.so, 10): Symbol not found: _build_reloptions
  Referenced from: /Applications/Postgres.app/Contents/Versions/12/lib/postgresql/pgroonga.so
  Expected in: /Applications/Postgres.app/Contents/Versions/12/bin/postgres
 in /Applications/Postgres.app/Contents/Versions/12/lib/postgresql/pgroonga.so
`

**Comment by kou** [comment]
What PostgreSQL version did you use to build PGroonga?
It seems that the latest PostgreSQL's formula is 13 not 12: https://github.com/Homebrew/homebrew-core/blob/master/Formula/postgresql.rb

**Comment by Somnium99** [comment]
when I run 'psql', it prints :
`psql (13.1, server 12.5)`
and if I run 'psql -c "\echo :VERSION"' it prints :
`PostgreSQL 13.1 on x86_64-apple-darwin20.1.0, compiled by Apple clang version 12.0.0 (clang-1200.0.32.27), 64-bit`

so it seems like 13, but if I go to `/Applications/Postgres.app/Contents/Versions/`, I see two folders : `12`, and `latest`.
The 'latest' folder however seems to be a redirect folder to the '12' folder.

OS version I'm using is MacOS 11.0.1 Big Sur.
This problem happens to me only on mac by the way. I've tried on windows and ubuntu 20 and they both work perfectly fine.

**Comment by kou** [comment]
Could you show the output of `psql -c '\set'`?

**Comment by kou** [comment]
Or `psql -c 'select version()'`.

**Comment by kou** [comment]
I think `\set` includes `SERVER_VERSION_NAME = '12.5 ...'` or something and `select version()` also returns `12.5 ...` or something.

You need to upgrade your PostgreSQL 13 from 12.
Could you upgrade your PostgreSQL to 13 and try again?

**Comment by Somnium99** [comment]
```
❯ psql -c '\set'
AUTOCOMMIT = 'on'
COMP_KEYWORD_CASE = 'preserve-upper'
DBNAME = 'my_name'
ECHO = 'none'
ECHO_HIDDEN = 'off'
ENCODING = 'UTF8'
FETCH_COUNT = '0'
HIDE_TABLEAM = 'off'
HISTCONTROL = 'none'
HISTSIZE = '500'
HOST = '/tmp'
IGNOREEOF = '0'
LAST_ERROR_MESSAGE = ''
LAST_ERROR_SQLSTATE = '00000'
ON_ERROR_ROLLBACK = 'off'
ON_ERROR_STOP = 'off'
PORT = '5432'
PROMPT1 = '%/%R%x%# '
PROMPT2 = '%/%R%x%# '
PROMPT3 = '>> '
QUIET = 'off'
SERVER_VERSION_NAME = '12.5'
SERVER_VERSION_NUM = '120005'
SHOW_CONTEXT = 'errors'
SINGLELINE = 'off'
SINGLESTEP = 'off'
USER = 'my_name'
VERBOSITY = 'default'
VERSION = 'PostgreSQL 13.1 on x86_64-apple-darwin20.1.0, compiled by Apple clang version 12.0.0 (clang-1200.0.32.27), 64-bit'
VERSION_NAME = '13.1'
VERSION_NUM = '130001'
```

and for the second one :
```
❯ psql -c 'select version()'
                                                   version                                                   
-------------------------------------------------------------------------------------------------------------
 PostgreSQL 12.5 on x86_64-apple-darwin16.7.0, compiled by Apple LLVM version 8.1.0 (clang-802.0.42), 64-bit
(1 row)

```

**Comment by Somnium99** [comment]
I tried reinstalling postgres with brew, but it still shows it as 12.5 when I use `select version()`. Do you know perhaps how to upgrade it to 13?

Also, VERSION_NAME seems to be 13.1 , and only SERVER_VERSION_NAME is 12.5. What's the difference? Could it be that SERVER_VERSION_NAME is only upgradable up to 12.5 ?

**Comment by kou** [comment]
`VERSION` is `psql`'s version not PostgreSQL's version.

How did you install your PostgreSQL?
If you installed your PostgreSQL by Homebrew, it's installed at `brew --prefix postgresql` not `/Applications/Postgres.app/...`.

**Comment by kou** [comment]
https://postgresapp.com/ ?

**Comment by Somnium99** [comment]
Ok , it's probably something is mixed with both Postgresapp and brew installs. 
I tried installing the latest Postgres.app (13) but it didn't work, it says binary files are missing, and the title inside the app showed "12" instead of 13.
So I removed it altogether, completely.
And now trying to solve it using only the brew installation.

Now I can see that :
` psql --version 
psql (PostgreSQL) 13.1` 

But when I run `psql -c 'select version()'`, I get: 

`psql: error: FATAL:  database "my_user_name" does not exist`.
I'm trying to solve it.. I'm hopeful that once I do, it will make pgroonga work. I'll update regarding it
